guestmetric/guestmetric_linux.go

```diff
@@ -199,6 +199,9 @@ func (c *Collector) CollectDisk() (GuestMetric, error) {
 				}
 			}
 			real_dev := ""
+			if !strings.Contains(disk, "xv") {
+				continue
+			}
```

Comment:
I would like to suggest to use more grace method like:
if c.Client != nil {
  				nodename, err := readSysfs(fmt.Sprintf("/sys/block/%s/device/nodename", disk))
  				if err == nil {
 					backend, err := c.Client.Read(fmt.Sprintf("%s/backend", nodename))
 				if err != nil {
 					return nil, err
 				}
 				real_dev, err = c.Client.Read(fmt.Sprintf("%s/dev", backend))
 				if err != nil {
 					return nil, err
 				}
 				}
 				
 			}